### PR TITLE
[docs] Add small examples to the Functional-Style Selectors topic

### LINKED ISSUE #2538: [docs] Add small examples to the Functional-Style Selectors topic
### Are you requesting a feature or reporting a bug?

An enhancement

### What is the current behavior?

The [Functional-Style Selectors](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.html) topic has combined examples each showing multiple methods.

### What is the expected behavior?

We need small examples for each method and put them right after the method description.

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @kirovboris @helen-dikareva 

closes #2538

### CODE REVIEW

File: docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md [hunk 1]
```diff
@@ -159,6 +228,20 @@ Method | Description
 `parent(cssSelector)` | Finds all parents of all nodes in the matching set and filters them by `cssSelector`.
 `parent(filterFn, dependencies)` | Finds all parents of all nodes in the matching set and filters them by the `filterFn` predicate. Use an optional `dependencies` parameter to pass functions, variables or objects to the `filterFn` function. See [Filtering DOM Elements by Predicates](#filtering-dom-elements-by-predicates).
 
+```js
+// Selects all ancestors of all ul elements.
+Selector('ul').parent();
```
Comment (kirovboris) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md]: In fact, we get the first ul and its first parent. Although we use "all parents" in the description of method, here we describe the specific element selector. I think for others examples it should be clarified as well.

File: docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md [hunk 2]
```diff
@@ -66,25 +92,62 @@ Parameter                     | Type                 | Description
 
 If `attrName` or `attrValue` is a String, `withAttribute` selects an element by strict match.
 
+```js
+// Selects div elements whose 'attrName' attribute
+// is set to 'foo'.
+// Does not match the 'attrName1' attribute or the 'foobar' value.
+Selector('div').withAttribute('attrName', 'foo');
```
Comment (kirovboris) [inline docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md]: IMO, `or the 'foobar' value.` looks weird in this sentence. 
I'd rephrase it like "Does not match the 'attrName1' attribute or the `attrName` attribute with the 'foobar' value."

File: docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md (hunk 1 above)
Comment (helen-dikareva) [reply docs/articles/documentation/test-api/selecting-page-elements/selectors/functional-style-selectors.md]: I disagree, this selector will work strictly in described way. In all examples above we say: `Selects all elements which ....`, because nowhere we use `nth(0)`. 
Probably we should have little topic with information about difference between:
`const allDivElements = Selector('div')` and `const firstDivElement = await Selector('div')`, I've already faced with questions about it. This conception is not obvious for some users because requires deep level of Selector understanding.
